packages/elements/src/overlay-menu/index.ts

```diff
@@ -378,6 +390,10 @@ export class OverlayMenu extends Overlay {
   protected update (changedProperties: PropertyValues): void {
     if (changedProperties.has('opened')) {
       if (this.opened) {
+        const parentMenuItem = OpenedMenusManager.getParentMenuItem(this);
+        if (parentMenuItem) {
+          parentMenuItem.setAttribute('aria-expanded', 'true');
```

Comment:
Would it be better to write it like this instead,
```suggestion
          parentMenuItem.setAttribute('aria-expanded', String(this.opened));
```

Reply:
It always set to "true" in case this.opened is equal true.